Makefile

```diff
@@ -22,3 +41,14 @@ push:
 
 	docker push digitalmarketplace/base-frontend:${VERSION}
 	docker push digitalmarketplace/base-frontend:latest
+
+.PHONY: push-py2
+push-py2:
+	docker push digitalmarketplace/base:${PY2VERSION}
+	docker push digitalmarketplace/base:latest
```

Comment:
If we drop these `:latest` pushes from either PY2 or PY3 jobs we can always know what they refer to independent of the job order. Since we should be pinning the version everywhere I'd say go with PY3 latest.